zkevm-circuits/src/evm_circuit.rs

```diff
@@ -365,12 +365,14 @@ pub mod test {
         run_test_circuit(block)
     }
 
-    pub fn run_test_circuit<F: Field>(block: Block<F>) -> Result<(), Vec<VerifyFailure>> {
```

Comment:
previous public function did not have a documentation comment, and clippy didn't give a warning, I guess because the module is called test? So, I didn't document the other test functions